common/djangoapps/student/tests/test_views.py

```diff
@@ -777,7 +777,11 @@ def test_content_gating_course_card_changes(self):
             user=self.user,
             course_id=course.id
         )
-        schedule = ScheduleFactory(start=self.THREE_YEARS_AGO + timedelta(days=1), enrollment=enrollment)
+        schedule = ScheduleFactory(
+            start=self.THREE_YEARS_AGO + timedelta(days=1),
```

Comment:
Unlikely but what if somehow the execution of this spans 2 days? We should calculate the delta once and use that variable for the of the ScheduleFactory.

Reply:
Ok, will change this to use variable. 
I didn't change it as this change was temporary since 'start' will be removed in next release.